### ISSUE TITLE
Does django-tenants support replication ?

### ISSUE THREAD

**Issue by thelinuxer** [tomturner/django-tenants#122]
Hi, 

Having all tenants in a single database is more reason to support replication to avoid bottlenecks. From the code I can't see how possibly you could be handling routing to different databases based on reads and writes. From what I can see the database wrapper is set only for the default connection when connection.set_schema connection.set_tenant is called, am I right ? In that case it's expected to fail to read from the read replica since the schema is only set on the default database.

Are you planning to support this feature in the near future ? We can try to find a solution for it together.

I assume the consumer would have configuration like the following:

A very simple router:
```python
class PrimaryReplicaRouter(object):
    def db_for_read(self, model, **hints):
        """
        Reads go to a replica.
        """
        return 'replica'

    def db_for_write(self, model, **hints):
        """
        Writes always go to primary.
        """
        return 'default'
```

and in the settings.py, the DATABASES configuration:

```python
DATABASES = {
    'default': {
        'ENGINE': 'django_tenants.postgresql_backend',
        'NAME': 'userdb',
        'USER': 'master',
        'PASSWORD': 'password',
        'HOST': 'localhost',
        'PORT': 5432,
        'ATOMIC_REQUESTS': True
    },
    'replica': {
        'ENGINE': 'tenant_backend',
        'NAME': 'userdb',
        'USER': 'master',
        'PASSWORD': 'password',
        'HOST': 'localhost',
        'PORT': 5433,
        'ATOMIC_REQUESTS': True
    },
}

DATABASE_ROUTERS = (
    'django_tenants.routers.TenantSyncRouter',
    'replicationrouter.PrimaryReplicaRouter',
)
```

**Comment by tomturner** [comment]
@thelinuxer Happy to have a PR for this

**Comment by jcass77** [comment]
Should this issue be closed now that #179 has been merged?

**Comment by mmcclelland1002** [comment]
@tomturner @jcass77 Made a PR for this feature: https://github.com/django-tenants/django-tenants/pull/760
Please review whenever you have availability. Thanks! :)
